### DISCUSSION TITLE
Inconsistent loading time when loading patterns into spaCy from FastAPI

### DISCUSSION THREAD

**Discussion by raqibhayder** [fastapi/fastapi#8860] [Questions] [answered]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [x] I already read and followed all the tutorial in the docs and didn't find an answer.
* [x] I already checked if it is not related to FastAPI but [Pydantic](https://github.com/samuelcolvin/pydantic).
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications, and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

### Example

I have created a repo called [fruits](https://github.com/raqib-hayder/fruits) which is self-contained and reproduces the issue at hand. 

### Description
I was using `flask` and `spaCy` to serve a named entity recognition (NER) model to detect entities in text, very similar to the [fruits](https://github.com/raqib-hayder/fruits) repo.  I load patterns from two pattern files, `fruits.jsonl` and `animals.jsonl`, which would take about 1 minute or so to load the ~40k patterns.  

I decided to move from `flask` to `fastapi`, but unfortunately, this leads to pattern loading times going up and becoming inconsistent. The loading times for different scenarios are in the [README.md](https://github.com/raqib-hayder/fruits/blob/main/README.md) in the [fruits](https://github.com/raqib-hayder/fruits) repo. 

**History**

I first raised this issue in spaCy discussion [here](https://github.com/explosion/spaCy/discussions/7774) as it was thought to be a spaCy issue, but upon further investigation, I believe that it is a FastAPI issue. Using spaCy to load patterns from a standalone script does not slow down the load time. The slowdown only happens when patterns are being loaded inside a FastAPI application. 

Any help will be appreciated. Please ask any questions that you might have (I will be actively monitoring this issue).

### Environment

* OS:  macOS
* FastAPI Version [e.g. 0.65.0]:
* IDE: PyCharm
* FastAPI version: 0.63.0
* Python version: 3.7

**Comment by Kludex** [comment, marked as answer]
It's not a FastAPI issue. I'll continue this on the spaCy discussion.
